gbpservice/neutron/nsf/configurator/api/config.py

```diff
@@ -0,0 +1,16 @@
+# Server Specific Configurations
```

Comment:
yes, But we need to change path of root controller and module in this file according to our requirement.